frontend/templates/component.html

```diff
@@ -4,63 +4,74 @@
 </head>
 <body class="body" onload="init()">
-<header id="header" class="header w-clearfix"><img src="{{ url_for('static', filename='images/logo.png') }}"
+
+<header id="header" class="header"><img src="{{ url_for('static', filename='images/logo.png') }}"
                                                    loading="lazy" alt="" class="logo">
-    <div class="navigation w-clearfix">
+    <div class="navigation">
         <a href="/" class="navigation-links">Home</a>
-        <div class="navigation-links">&gt;</div>
+        <div class="navigation-link-separator">&gt;</div>
         <a href="#" class="navigation-links">Einzelkomponente</a>
     </div>
+
 </header>
-<div class="component-overview">
-    <div id="component-overview">
-        <div class="view-description-label">Einzelkomponente</div>
-        <div class="component-form-block w-form">
-            <form id="component-form" name="component-form" data-name="component-form" class="component-form">
-                <div class="description-columns w-row">
-                    <div class="default-column w-clearfix w-col w-col-6">
-                        <label for="component-name-2" class="entry-label">Name der Komponente</label>
-                        <input type="text" class="textfield w-input" maxlength="256" name="component-name"
-                               data-name="component-name" id="component-name">
-                        <input type="text" class="textfield w-input" maxlength="32" name="component-uid"
-                               data-name="component-uid" id="component-uid" disabled>
-                        <label for="component-category-2" class="entry-label">Komponenten-Kategorie</label>
-                        <select id="component-category" name="component-category" data-name="component-category"
-                                class="category-dropdown w-select" onchange="setSections(this.value)">
-                            <option value="default">Bitte auswählen..</option>
-                        </select></div>
-                    <div class="default-column w-clearfix w-col w-col-6"><label for="component-description-textarea-2"
-                                                                                class="entry-label">Beschreibung der
-                        Komponente</label><textarea name="component-description-textarea" maxlength="5000"
-                                                    id="component-description-textarea"
-                                                    data-name="component-description-textarea"
-                                                    class="textarea w-input"></textarea></div>
+
+<div class="component-overview" id="component-overview">
+
+    <div class="view-description-label">Einzelkomponente</div>
+
+    <div class="component-form-block">
+        <form id="component-form" name="component-form" data-name="component-form" class="component-form">
```

Comment:
Warum sind hier id, name und class vergeben?

Reply:
Ist in der Tat unnötig, wird entfernt.